api/app/admin/office.py

```diff
@@ -106,6 +106,10 @@ def get_query(self):
             ("0", 'Off - Hide Currently Waiting at bottom in Smartboard'), \
             ("1", 'On - Show Currently Waiting  from bottom in Smartboard')
         ],
+        'optout_status': [
```

Comment:
Just to be really clear, can we please change the labels on the options to read:

- Off - Open tickets automatically closed at end of day
- On - Tickets must be closed manually